ui/app/manager/src/pages/page-configuration.ts

```diff
@@ -318,7 +318,8 @@ export class PageConfiguration extends Page<AppStateKeyed> {
         // POST manager config if changed...
         if (this.managerConfigurationChanged) {
             managerPromise = manager.rest.api.ConfigurationResource.update(config).then(() => {
-                fetch(this.urlPrefix + "/manager_config.json", {cache: "reload"});
+                // Not sure what this does and how to adapt it to fetch from axios, so I just replaced the urlPrefix with the managerUrl
+                fetch(this.managerConfiguration.manager.managerUrl + "/api/master/configuration/manager", {cache: "reload"});
```

Comment:
Result is not used and anyway why is this needed? The ConfigurationResource.update should return the updated configuration and this should be used.